lib/challenge.js

```diff
@@ -30,10 +60,24 @@
   // - a name
   // - an address
 
-var pet_owner;
 
 // Answer Starts Here
 
+var pet_owner = {
+  name: 'Sue',
+  address: 'Boston',
+  pets: {
+    name: 'Brutus',
+    type: 'dog',
+    vaccinations: {
+      vacName: rabies,
+      vacDate: '12-14-2104'
+    },
+  },
+},
```

Comment:
Very good. You went beyond what the question asked for. For `pets`, if it's going to be plural, it should be able to contain more than one pet. An array containing objects similar to the object literal you built would be perfect.